The installsw seems to have problems with long lines, so we wrap
each of the potentially long lines using fmt.

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

--- oe-core/meta/classes/package_rpm.bbclass.orig	2013-01-22 18:41:31.112166910 -0600
+++ oe-core/meta/classes/package_rpm.bbclass	2013-01-22 18:41:00.600166377 -0600
@@ -452,7 +452,7 @@
 		pkgs_to_remove=`cat ${target_rootfs}/install/tmp/remove.list | xargs echo`
 		pkgs_to_install=`cat ${target_rootfs}/install/tmp/install.list | xargs echo`
 		
-		echo "Note: to be removed: ${pkgs_to_remove}"
+		echo "Note: to be removed: ${pkgs_to_remove}" | fmt -s
 
 		for pkg in ${pkgs_to_remove}; do
 			echo "Debug: What required: $pkg"
@@ -462,12 +462,12 @@
 		[ -n "$pkgs_to_remove" ] && smart --data-dir=${target_rootfs}/var/lib/smart remove -y ${pkgs_to_remove}
 	fi
 
-	echo "Note: to be installed: ${pkgs_to_install}"
+	echo "Note: to be installed: ${pkgs_to_install}" | fmt -s
 	[ -n "$pkgs_to_install" ] && smart --data-dir=${target_rootfs}/var/lib/smart install -y ${pkgs_to_install}
 
 	if [ -n "${package_attemptonly}" ]; then
 		echo "Note: installing attempt only packages..."
-		echo "Attempting $pkgs_to_install"
+		echo "Attempting $pkgs_to_install" | fmt -s
 		echo "Note: see `dirname ${BB_LOGFILE}`/log.do_${task}_attemptonly.${PID}"
 		translate_oe_to_smart ${sdk_mode} --attemptonly $package_attemptonly
 		for each_pkg in $pkgs_to_install ;  do
